experanto/dataloaders.py

```diff
@@ -0,0 +1,17 @@
+from typing import Any, List, Optional, Tuple
+from omegaconf import DictConfig
+from torch.utils.data import DataLoader
+
+from .datasets import ChunkDataset
+from .utils import MultiEpochsDataLoader, LongCycler
+
+
+def get_multisession_dataloader(paths, config: DictConfig) -> DataLoader:
+    dataloaders = {}
+    for i, path in enumerate(paths):
+        dataset_name = path.split("dynamic")[1].split("-Video")[0]
```

Comment:
this would limit the name folders, why just not to take the whole folder name? Then it would not have to contain "dynamic", etc in it and it might be easier to integrate Allen brain dataset, for instance